tests/tick.test.mjs

```diff
@@ -727,15 +727,17 @@ describe('Tick', async () => {
         memory: zeroPeriodDemandFactorTick.memory,
         timestamp: demandFactorSettings.periodZeroStartTimestamp,
       });
-    assert.equal(baseFeeAtFirstDemandFactorPeriod, 600_000_000);
+    assert.equal(baseFeeAtFirstDemandFactorPeriod, 1_200_000_000);
 
     let tickMemory = zeroPeriodDemandFactorTick.memory;
 
     // compute the periods until we get to 0.5 from the current demand factor
-    const periodsUntilMinDemandFactor = Math.ceil(
-      Math.log(demandFactorSettings.demandFactorMin) /
-        Math.log(1 - demandFactorSettings.demandFactorDownAdjustmentRate),
-    );
+    const periodsUntilMinDemandFactor =
+      Math.ceil(
+        Math.log(demandFactorSettings.demandFactorMin) /
+          Math.log(1 - demandFactorSettings.demandFactorDownAdjustmentRate),
```

Comment:
nit: this 1 can be the DemandFactor.currentDemandFactor. That would remove the need for the * 2 and should work with any initial demand factor value